src/redux/books/books.js

```diff
@@ -0,0 +1,18 @@
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+export default function booksReducer(state = { books: [] }, action = {}) {
+  switch (action.type) {
+    case ADD_BOOK: return state.books.push(action.book);
+    case REMOVE_BOOK: return state.books.pop(action.book);
+    default: return state;
+  }
+}
+
+export function addBook(book) {
+  return { type: ADD_BOOK, book };
+}
+
+export function removeBook(book) {
+  return { type: REMOVE_BOOK, book };
```

Comment:
- [ ] The requirements of this project state that: in case `action.type` is `REMOVE_BOOK`, the function should use` ES6 filter()` method to create a new array, which will not contain the book you want to remove from the store. It will be nice if you can filter by the book id key. Please kindly fix this and move a step forward towards approval.